src/main/java/org/dependencytrack/notification/publisher/PublisherClass.java

```diff
@@ -0,0 +1,13 @@
+package org.dependencytrack.notification.publisher;
```

Comment:
Dependency track code documentation should also be added to the top like other files: https://github.com/DependencyTrack/dependency-track/blob/master/src/main/java/org/dependencytrack/notification/publisher/Publisher.java#L1